pdns/recursor_cache.hh

```diff
@@ -146,7 +146,7 @@ private:
 
     Netmask lookupBestMatch(const ComboAddress& addr) const
     {
-      const auto best = d_nmt.lookup(addr);
+      const auto * best = d_nmt.lookup(addr);
```

Comment:
Wondering what the point is here, did clang-tidy suggest this?

Reply:
> readability-qualified-auto: `const auto best` can be declared auto * const best`. 

And also one of my error, that I always do in cpp misplaced the `const` in my fix. :facepalm:

Reply:
Pushed something to follow exactly clang suggestion